libafl_frida/src/executor.rs

```diff
@@ -102,13 +102,13 @@ where
         if self.helper.stalker_enabled() {
             self.stalker.deactivate();
         }
-        #[cfg(unix)]
-        unsafe {
-            if ASAN_ERRORS.is_some() && !ASAN_ERRORS.as_ref().unwrap().is_empty() {
-                log::error!("Crashing target as it had ASAN errors");
-                libc::raise(libc::SIGABRT);
-            }
-        }
+        // #[cfg(unix)]
```

Comment:
This is commented out only for testing?

Reply:
Yes, and now. I actually wanted to use continue_on_error, but in this place, it was ignoring this option and always crashing the process. Is that on purpose?

Reply:
IIRC, continue_on_error will error out here, becuase we want to be able to trap all asan errors for this test case, but if we trapped any asan errors we need to report it here.
We can change this behavior, I guess.

Reply:
What do you suggest?

Reply:
I'm not sure what to do. Perhaps we need to attach the `AsanError`s to the test-case's metadata, but not necessarily crash. 

Or alternatively, add a testing specific cfg.

Reply:
Done (test-specific cfg)